rpm/update-config-files.sh

```diff
@@ -17,25 +17,19 @@ ensure() {
     then
       sed -i "/^#.*$2[[:blank:]].*=/a \
 $2 = $3" $1
-    else
-      echo "$2 = $3" >> $1
     fi
```

Comment:
The description says “fail if a pattern is not found”; how does that happen? I don’t see, in my tests, that `if false …` returns a false status.